### ISSUE TITLE
Directionality of Connections can be confusing

### ISSUE THREAD

**Issue by pkgw** [pubpub/pubpub#1539]
### What's your enhancement request?

I was just adding a Connection to a pub and had trouble figuring out which direction to use. In particular, I was adding an Arxiv preprint link. Depending on the directionality, the label is either:

> A Preprint of this Pub

or 

> This Pub is a Preprint of

The second of these is pretty unambiguous, I think, but I found the first hard to parse. I first read it as "[The current Pub is] A Preprint of this Pub[:]", and got confused because it seemed like both options meant the same thing.

### What would this enhancement help you accomplish?

Get the correct direction when making Connections.

---

I haven't spent much time thinking about this, but the first idea I had was that maybe the first option could be presented as "This Pub has a Preprint:".

**Comment by gabestein** [comment]
Thanks for these, Peter. I agree we need to do better here, and we've really struggled with the right language. I like your example, and we'll discuss on our next prio call!
